shell/platform/android/io/flutter/embedding/android/FlutterView.java

```diff
@@ -69,6 +71,8 @@
   @Nullable
   private FlutterRenderer.RenderSurface renderSurface;
   private boolean didRenderFirstFrame;
+  @Nullable
+  private PlatformViewsController platformViewsController;
```

Comment:
I understand why one is given to the flutterview on attachToFlutterEngine but I'm not sure I understand why the flutterview instance then holds onto it.

Reply:
How would we call `detachAccessibilityBridge()` without a reference to the `PlatformViewsController`?

Reply:
Talked offline, resolving old comments.